plugins/labhub.py

```diff
@@ -279,12 +279,14 @@ def difficulty_level(user, iss):
                 2. The user belongs to developers or maintainers team as well as
                    newcomers team.
                 False if
-                1. A newcomer asks for assignment to an issue that has no
+                1. A member asks for assignment to an issue that has 
+                   initiatives/gci label.
+                2. A newcomer asks for assignment to an issue that has no
                    difficulty label.
-                2. A newcomer asks for assignment to an issue with difficulty
+                3. A newcomer asks for assignment to an issue with difficulty
                    higher than low.
                 """
-                if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user)
+                elif (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user)
```

Comment:
Line is longer than allowed. (81 > 80)

*Origin: LineLengthBear, Section: `all.linelength`.*